tests/hurl/accounts.hurl

```diff
@@ -13,8 +13,50 @@ query Accounts {
 ```
 HTTP 200
 [Asserts]
-duration < 1000
+
 jsonpath "$.data.accounts[0].publicKey" == "B62qj8KB2fk59NkV4VuoTkVXHjw8VJzC3ybKrWo7zuDC9xTiWXPygEe"
 jsonpath "$.data.accounts[0].balance" == 719020000000
 jsonpath "$.data.accounts[0].nonce" == 1
 jsonpath "$.data.accounts[0].delegate" == "B62qj8KB2fk59NkV4VuoTkVXHjw8VJzC3ybKrWo7zuDC9xTiWXPygEe"
+jsonpath "$.data.accounts[0].timeLocked" == false
+
+duration < 100
+
+POST {{url}}
+```graphql
+query Accounts {
+  accounts(query: {publicKey: "B62qoTaYnDYrtmver4jnkgJbFU6iohjtALTU4ebf4yCKrxFFHd9Dimh" }) {
```

Comment:
If we only query by public key, will the response values change??

Reply:
Same value, we just get it a faster way now